### ISSUE TITLE
Default startup YSF Reflector

### ISSUE THREAD

**Issue by iw2gob** [g4klx/YSFClients#101]
Hi,
in this version of YSFGateway the default startup of a YSF reflector no longer works; therefore it is no longer possible to change YSF reflector from the dashboard.

73, Maurizio IW2GOB

**Comment by AndyTaylorTweet** [comment]
I have the same, setting the Startup reflector to anything causes the process to crash for me.

**Comment by iw2gob** [comment]
Yes Andy, same here. For now I prefer to go back to the previous version. 73, Maurizio IW2GOB

**Comment by AndyTaylorTweet** [comment]
@iw2gob 
Yesterday's commit sorted out the issue of the process bailing out when a startup reflector is listed in the config, the format of the startup reflector has changed too.

**Comment by iw2gob** [comment]
That's right Andy !
